build/deps/shared_deps.jsonc

```diff
@@ -49,23 +49,23 @@
       "owner": "astral-sh",
       "repo": "ruff",
       "file_regex": "^ruff-aarch64-apple-darwin.tar.gz$",
-      "build_file_content": "filegroup(name='file', srcs=glob(['**']))"
+      "build_file_content": "filegroup(name='file', srcs=['ruff'], visibility=['//visibility:public'])"
```

Comment:
Nit: I think it's cleaner to continue to still use a glob and capture all files here, the actual name of the binary feels like an implementation detail here that the dependency definition doesn't need to know about.

Reply:
If we keep using a glob then copy_file fails on there being more than one file 😢